doc/troubleshooting.rst

```diff
@@ -107,14 +107,25 @@ Log Files
 
 Effectively debugging software requires as much relevant information as can be
 obtained.  To assist the ownCloud support personnel, please try to provide as
-many relevant logs as possible. Log output can help  with tracking down
+many relevant logs as possible. Log output can help with tracking down
 problems and, if you report a bug, log output can help to resolve an issue more
 quickly.
 
+The client log file is often the most helpful log to provide.
+
 Obtaining the Client Log File
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To obtain the client log file:
+There are several ways to produce log files. The most commonly useful is enabling
+logging to a temporary directory, described first.
+
+Note: Client log files contain file and folder names, metadata, server urls and
+other private information. Only upload them if you are comfortable sharing the
+information. Logs are often essential to tracking down a problem though, so
```

Comment:
I think either: 

- to track down 
- when tracking down

... sound better. cc/ @tomneedham for native speaker kung-fu.

Reply:
I'll adjust to "for tracking down a problem"